code-challenges.test.js

```diff
@@ -43,8 +91,63 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+describe("capitalArrays", () =>{
+  it("takes in an array of words and returns an array with all the words capitalized", () => {
+    expect(capitalArrays(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(capitalArrays(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+  })
+})
+// Failing!
+//  FAIL  ./code-challenges.test.js
+  // ● Test suite failed to run
+
+  // ReferenceError: capitalArray is not defined
+
 // b) Create the function that makes the test pass.
 
+// Pseudocode:
+// input: 2 arrays
+// output: 2 arrays with words capitalized
+
+// Process:
+// Create a function named capitalArrays that takes in an array of words and returns an array with all the words capitalized
+// use .join() to keep the words in one array together
+// Use .charAt(0) to capitalize the words in the arrays
+
+// FUNCTION
+const capitalArrays = (array1) => {
+  return array1.join(" ").charAt(0).split(" ")
```

Comment:
One thing to keep in mind is that join will work on an array and convert it to a string. I like the idea of using charAt to single out the first letter of each word, this just need a bit more. Because we have an array, an approach we could take would be to use a HOF to iterate over each element (string) and then perform the logic such as for charAt (0) I want to uppercase that letter and then join in with the rest of the word.